model/custom_status.go

```diff
@@ -33,6 +45,23 @@ func (cs *CustomStatus) ToJson() string {
 	return string(b)
 }
 
+func (cs *CustomStatus) IsDurationValid() bool {
+	switch cs.Duration {
+	case DontClear, ThirtyMinutes, OneHour, FourHours, Today, ThisWeek, CustomDateTime:
+		return true
+	}
+
+	return false
+}
+
+func (cs *CustomStatus) IsExpirationTimeValid() bool {
+	if cs.Duration != DontClear && (cs.ExpiresAt.IsZero() || cs.ExpiresAt.Before(time.Now())) {
+		return false
+	}
+
+	return true
```

Comment:
Don't use if/else for return true/false

```suggestion
	return !(cs.Duration != DontClear && (cs.ExpiresAt.IsZero() || cs.ExpiresAt.Before(time.Now()))) 
```